tests/integration/targets/podman_image/tasks/main.yml

```diff
@@ -40,6 +59,48 @@
           - pull5 is not changed
           - "'alpine-sh' in images.stdout"
           - "'library/alpine' in images.stdout"
+          - alpine_docker_archive.stat.exists
+
+    - name: Push to a tarball in the docker-archive format
+      containers.podman.podman_image:
+        name: 'docker.io/library/alpine'
+        push: true
+        push_args:
+          dest: '/tmp/alpine.tar'
+          transport: 'docker-archive'
+
+    - name: Stat the docker-archive tar file
+      stat:
+        path: '/tmp/alpine.tar'
+      register: alpine_docker_archive
+
+    - name: Cleanup docker-archive transport test file
+      ansible.builtin.file:
+        path: '/tmp/alpine.tar'
+        state: absent
+
+    - name: Remove local archive testing directory (pre-test)
+      ansible.builtin.file:
+        path: '/tmp/alpine'
+        state: absent
+
+    - name: Push to a local directory
+      containers.podman.podman_image:
+        name: 'docker.io/library/alpine'
+        push: true
+        push_args:
+          dest: '/tmp/alpine'
+          transport: 'dir'
+
+    - name: Remove local archive testing directory (post-test)
+      ansible.builtin.file:
```

Comment:
Can we check that `/tmp/alpine` exists before deleting? Like it's done above for `register: alpine_docker_archive`

Reply:
I cleaned up the tests. I think I wrote them too late at night :)